CONTRIBUTING-gitflow.md

```diff
@@ -1,95 +1,149 @@
+
+<h3>Project Cloned in System</h3>
+<br><br>
+
+<li><h3>Add 'upstream' repo to list of remotes</h3></li><br>
+Keeping Your Fork Updated
+While it's not required, if you plan on making anything more than a fast patch, you'll want to keep your fork up to date by tracking the original "upstream" source you forked previously. You'll need to add a remote to accomplish this:<br><br>
+
+`git remote add upstream https:/github.com/meshery/meshery.git`
+<br><br>
 ("meshery" is used as the example repo. Be sure to reference the _actual_ repo you're contributing to e.g. "meshery-linkerd").
+<br>
+
+<li><h3>Verify the new remote named 'upstream'</h3></li>
+
+`git remote -v`
+<br>
+
+<li><h3>Fetch from upstream remote</h3</li>
+<br>
+You'll need to fetch the upstream repo's branches and newest commits to bring them into your repository whenever you wish to update your fork with the latest upstream changes:
+<br>
+
+`git fetch upstream`
+<br><br>
+
+<li><h3>View all branches, including those from upstream</h3></li>
+<br>
+
+`git branch -va`
+<br>
+
+<li><h3>Checkout your master branch and merge upstream</h3></li>
+<br>
+Now, checkout your master branch and merge it with the master branch of the upstream repo:
+<br>
+
+`git checkout master`<br>
+`git merge upstream/master`
+<br><br>
+If the local master branch has no unique commits, git will simply execute a fast-forward. However, if you've been making modifications to master (which, in the vast majority of circumstances, you shouldn't be - see the next section), you may run into issues. Always keep in mind the changes made upstream when doing so.
+<br>
+Your local master branch is now up to date with everything that has been changed upstream.
+<br><br>
+
+<li><h3>Create a Branch (to avoid conflicts)</h3></li>
+<br>
+It's essential to create a new branch whenever you start working on a new feature or bugfix. Not only is it a standard git workflow, but it also organises and separates your modifications from the main branch, allowing you to simply submit and manage several pull requests for each task you finish.
+<br>
+Follow the steps below to establish a new branch and begin working on it.
+<br><br>
+<li><h3>Check out the master branch; from which your new branch will be derived.</h3></li>
+
+`git checkout master`
+<br>
+
+<li><h3>Create a new branch</h3></li> (Give your branch a simple, informative name.)
+<br>
+For enhancements use
+
+`feature/your_username/issue#` or `feature/your_username/name_of_feature`
+<br>
+For bugs use
+
+`bug/your_username/issue#` or `bug/your_username/name_of_bug`
+<br>
+
+`git branch feature/jdoe/567`
+<br><br>
+
+<li><h3>Activate your new branch.</h3></li>
+<br>
+
+`git checkout feature/jdoe/567`
+(Use the name of the branch you created instead of 'feature/jdoe/567'.)
+<br>
+
+Now you may start hacking and make any changes you desire.🚀
+<br><br>
+
+<li><h3>Stage the Changes</h3></li>
+
+`git add .`
+<br>
+(This will stage all the changes you have made.)
+<br>
```

Comment:
@Nandani-Paliwal I agree using `.` is faster, but for new contributors it might cause them to introduce unintended changes accidentally so can we change the `.` for `[files-changed]` please?